liwords-ui/src/lobby/active_games.tsx

```diff
@@ -40,9 +43,9 @@ export const ActiveGames = (props: Props) => {
 
   const handleChange = useCallback(
     (
-      pagination: TablePaginationConfig,
+      _pagination: TablePaginationConfig,
       filters: Record<string, FilterValue | null>,
-      sorter:
+      _sorter:
```

Comment:
would like to understand a little bit better why these are underscored -- `_pagination` and `_sorter`.

Reply:
becuase it's not being used.